doc/classes/KinematicBody2D.xml

```diff
@@ -7,6 +7,7 @@
 		Kinematic bodies are special types of bodies that are meant to be user-controlled. They are not affected by physics at all (to other types of bodies, such a character or a rigid body, these are the same as a static body). They have however, two main uses:
 		Simulated Motion: When these bodies are moved manually, either from code or from an AnimationPlayer (with process mode set to fixed), the physics will automatically compute an estimate of their linear and angular velocity. This makes them very useful for moving platforms or other AnimationPlayer-controlled objects (like a door, a bridge that opens, etc).
 		Kinematic Characters: KinematicBody2D also has an API for moving objects (the [method move_and_collide] and [method move_and_slide] methods) while performing collision tests. This makes them really useful to implement characters that collide against a world, but that don't require advanced physics.
+		See https://docs.godotengine.org/en/latest/tutorials/physics/physics_introduction.html#kinematicbody2d for examples of usage.
```

Comment:
Actually the `<tutorials>` section right below is specifically meant for this (with syntax `<link></link>`).

Reply:
Yes, but that doesn't actually display, and I really want to direct more new users to that tutorial.

Reply:
What do you mean that doesn't actually display?
![screenshot_20190116_220254](https://user-images.githubusercontent.com/4701338/51278295-80718900-19da-11e9-9a6a-e29215c8b4bd.png)

Reply:
Ah I see that they don't display in the online docs. That's a bug in the conversion script that needs to be fixed, we shouldn't work it around by putting links in the description instead of the dedicated links section :)

Reply:
I didn't mean to remove it, but to put it in the `<tutorials>` section which is meant for it.

And actually tutorial links are supported by the online docs, but it seems only some of them appear (e.g. http://docs.godotengine.org/en/latest/classes/class_inputmap.html#tutorials). Looks like those without trailing anchor are not processed properly, CC @PJB3005.

Reply:
Well, now that I know we're going to get it fixed, I think I'll just do a sweep through and add them where needed.

Reply:
Unless I made a mistake, the tutorials should've been untouched by my changes to the script.

Though seriously the way the handling of tutorials (parsing existing links as to figure out a correct way to link them) instead of just providing a version agnostic path is kinda ridiculous.

Reply:
@PJB3005 Yeah it seems to have been broken even before your changes, I just pinged you in hope that you could have a look at fixing it nevertheless :P

> Though seriously the way the handling of tutorials (parsing existing links as to figure out a correct way to link them) instead of just providing a version agnostic path is kinda ridiculous.

A relative path to `tutorials/physics/physics_introduction.html#kinematicbody2d` would be fine by me.